x/interchainstaking/keeper/zones.go

```diff
@@ -164,6 +164,14 @@ func (k *Keeper) GetZoneForDepositAccount(ctx sdk.Context, address string) *type
 	return nil
 }
 
+func (k *Keeper) GetZoneForWithdrawalAccount(ctx sdk.Context, address string) *types.Zone {
```

Comment:
```suggestion
func (k *Keeper) GetZoneForWithdrawalAccount(ctx sdk.Context, address string) Z(*types.Zone, bool) {
```

could also return a `found` bool

Reply:
Sure; I kept it consistent with the corresponding methods for the other accounts. Let's update all of them.